core-services/template-deprecation/_allowlist.yaml

```diff
@@ -706,6 +706,12 @@ templates:
     unknown_blocker:
       description: unknown
       jobs:
+        pull-ci-openshift-cluster-etcd-operator-master-e2e-aws-disruptive:
+          generated: true
+          kind: presubmit
+        pull-ci-openshift-cluster-etcd-operator-master-e2e-gcp-disruptive:
+          generated: true
+          kind: presubmit
```

Comment:
Please add these under DPTP-1775 above (`git checkout HEAD^^^ core-services/template-deprecation/_allowlist.yaml; make template-allowlist BLOCKER=DPTP-1775` should do the trick, too)

Reply:
Done